ci/requirements-3.5_DOC_BUILD.run

```diff
@@ -17,5 +17,4 @@ xlsxwriter
 sqlalchemy
 numexpr
 bottleneck
-statsmodels
```

Comment:
statsmodels is still used for an example in the docs (I think the pipe example), so should leave it